bin/deploy.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+PUBLISH=0
+CHEF_VER="latest"
```

Comment:
@knightsamar in ``build_image.sh`` we have an option for the script which sets ``-var "chef_version=$CHEF_VERSION"`` in the packer build command. How does that get set in the pipeline?